typescript/lib/services/service_context.ts

```diff
@@ -0,0 +1,129 @@
+import * as bindingsJsGrpc from '../../kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb'; //TODO - should I be importing all modules or just specific ones for best practices (need to check)
+import * as bindingsJs from '../../kurtosis_core_rpc_api_bindings/api_container_service_pb'; //TODO (comment) - I cannont merge with above line unfortunately, can only import one module at a time MAXIMUM
+import { ServiceID } from './service';
+import { StaticFileID } from './container_creation_config'; 
+import * as grpc from "grpc"; //TODO - this is needed for grpc callback, but this import is tentative 
+import * as constructors from "../constructor_calls";
+var path = require("path"); //TODO - I don't think this works
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+class GeneratedFileFilepaths { //TODO (comment) - try interface if class doesn't work
+	absoluteFilepathOnTestsuiteContainer: string;
+	absoluteFilepathOnServiceContainer:   string;
+}
+
+class ServiceContext {
+    
+    client: bindingsJsGrpc.ApiContainerServiceClient;
+    serviceId: ServiceID;
+	ipAddress: string;
```

Comment:
Looks like mismatched indentation here

Reply:
That is so odd since Visual Studio keeps on showing me that I have indented it properly

![Screen Shot 2021-07-23 at 11 05 56 AM](https://user-images.githubusercontent.com/52847195/126801906-30661d21-d812-4a79-a8c1-d7e68fb1e2e8.png)